CoverFragment.loadMediaInfo IAE

Version: 1.5.x

```
java.lang.RuntimeException: Error receiving broadcast Intent { act=action.de.danoeh.antennapod.core.service.playerStatusChanged flg=0x10 } in de.danoeh.antennapod.core.util.playback.PlaybackController$4@cd3dfb6
	at ...
Caused by: java.lang.IllegalArgumentException: You cannot start a load on a fragment before it is attached
	at com.bumptech.glide.Glide.with(Glide.java:4110)
	at de.danoeh.antennapod.fragment.CoverFragment.loadMediaInfo(CoverFragment.java:74)
	at de.danoeh.antennapod.activity.AudioplayerActivity$AudioplayerPagerAdapter.onMediaChanged(AudioplayerActivity.java:1110)
	at de.danoeh.antennapod.activity.AudioplayerActivity.loadMediaInfo(AudioplayerActivity.java:280)
	at de.danoeh.antennapod.activity.MediaplayerActivity$1.loadMediaInfo(MediaplayerActivity.java:141)
	at ...
```

--- a/app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java
+++ b/app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java
@@ -63,12 +63,7 @@ public class CoverFragment extends Fragment implements AudioplayerContentFragmen
     }
 
     private void loadMediaInfo() {
-        if(imgvCover == null) {
-            return;
-        }
         if (media != null) {
-            Log.d(TAG, "feed title: " + media.getFeedTitle());
-            Log.d(TAG, "episode title: " + media.getEpisodeTitle());
             txtvPodcastTitle.setText(media.getFeedTitle());
             txtvEpisodeTitle.setText(media.getEpisodeTitle());
             Glide.with(this)
@@ -103,7 +98,7 @@ public class CoverFragment extends Fragment implements AudioplayerContentFragmen
 
     @Override
     public void onMediaChanged(Playable media) {
-        if(this.media == media) {
+        if(!isAdded() || this.media == media) {
             return;
         }
         this.media = media;

--- a/app/src/main/java/de/danoeh/antennapod/fragment/ItemDescriptionFragment.java
+++ b/app/src/main/java/de/danoeh/antennapod/fragment/ItemDescriptionFragment.java
@@ -62,7 +62,6 @@ public class ItemDescriptionFragment extends Fragment implements AudioplayerCont
     private static final String ARG_HIGHLIGHT_TIMECODES = "arg.highlightTimecodes";
 
     private WebView webvDescription;
-    private String webvData;
 
     private ShownotesProvider shownotesProvider;
     private Playable media;
@@ -309,7 +308,6 @@ public class ItemDescriptionFragment extends Fragment implements AudioplayerCont
                 .subscribeOn(Schedulers.newThread())
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(data -> {
-                    webvData = data;
                     webvDescription.loadDataWithBaseURL(null, data, "text/html",
                             "utf-8", "about:blank");
                     Log.d(TAG, "Webview loaded");
@@ -384,7 +382,7 @@ public class ItemDescriptionFragment extends Fragment implements AudioplayerCont
 
     @Override
     public void onMediaChanged(Playable media) {
-        if(this.media == media) {
+        if(this.media == media || webvDescription == null) {
             return;
         }
         this.media = media;

